Update README.md (#7818)
ollama/ollama: README.md

- [Perfect Memory AI](https://www.perfectmemory.ai/) (Productivity AI assists personalized by what you have seen on your screen, heard and said in the meetings)
- [Hexabot](https://github.com/hexastack/hexabot) (A conversational AI builder)
- [Reddit Rate](https://github.com/rapidarchitect/reddit_analyzer) (Search and Rate Reddit topics with a weighted summation)

> - [OpenTalkGpt](https://github.com/adarshM84/OpenTalkGpt)  (First Chrome Extension: Manage open-source models supported by Ollama, create custom models, and chat with models—all from a user-friendly UI.)

Reviewer: ```suggestion
- [OpenTalkGpt](https://github.com/adarshM84/OpenTalkGpt) (Chrome Extension to manage open-source models supported by Ollama, create custom models, and chat with models from a user-friendly UI)
```
